core/templates/dev/head/collection_editor/collection_editor.html

```diff
@@ -0,0 +1,43 @@
+{% extends "base.html" %}
+
+{% block maintitle %}
+  Oppia Editor
+{% endblock maintitle %}
+
+{% block subtitle %}
+  {{ title }}
```

Comment:
I don't think you're sending the title to the frontend (through CollectionEditorPage).